pkg/users/local.js

```diff
@@ -92,7 +92,7 @@ function passwd_self(old_pass, new_pass) {
             dfd.resolve();
         })
         .fail(function(ex) {
-            if (ex.constructor.name == "ProcessError")
+            if (ex.constructor.name == "ProcessError" || ex.process_error)
```

Comment:
Why do we need this at all? We should probably just check for the presence of exit_status?